OSDOCS-16873-abstracts: SCALE-3: Low-Latency/CNF Provisioning and Tuning (#109052)
openshift/openshift-docs: snippets/worker-latency-profile-intro.adoc

:_mod-docs-content-type: SNIPPET

[role="_abstract"]

> If as a cluster administrator, you performed latency tests for platform verification, you can then discover the need to adjust the operation of the cluster to ensure stability in cases of high latency. 

Reviewer: Small suggestion: I feel like the original intent of this text might have been to use "can" to indicate possibility of something happening rather than ability, in which case "might" would be the better word choice:

```suggestion
If as a cluster administrator, you performed latency tests for platform verification, you might discover the need to adjust the operation of the cluster to ensure stability in cases of high latency. 
```
Up to you though

Author: Good point. Updated!
